atmo/coordinator/coordinator.go

```diff
@@ -21,15 +21,15 @@ import (
 )
 
 const (
-	atmoMethodSchedule       = "SCHED"
-	atmoMethodStream         = "STREAM"
-	atmoHeadlessStateHeader  = "X-Atmo-State"
-	atmoHeadlessParamsHeader = "X-Atmo-Params"
-	atmoRequestIDHeader      = "X-Atmo-RequestID"
-	atmoMessageURI           = "/meta/message"
-	AtmoMetricsURI           = "/meta/metrics"
-	AtmoHealthURI            = "/health"
-	connectionKeyFormat      = "%s.%s.%s"
+	atmoMethodSchedule = "SCHED"
+	atmoMethodStream   = "STREAM"
+	// atmoHeadlessStateHeader  = "X-Atmo-State"
+	// atmoHeadlessParamsHeader = "X-Atmo-Params"
+	// atmoRequestIDHeader      = "X-Atmo-RequestID"
```

Comment:
@javorszky I commented these out for the time being because they are unused and failing the lint. Should they be there though? @cohix?

Reply:
I'm of the opinion that unused code should not be present. Previously I wanted to remove these, and then @cohix stopped me. I'm team #removeUnusedCode

Reply:
Hmmmm. I could have sworn we are still using those 🤔

Reply:
Ya we aren't in this project, and they cannot be imported elsewhere, correct? What should we do for the time being. 

We could disable the unused linter, comment these out, or remove them.

Reply:
I want to remove them because they can't be used elsewhere, and they're not being used here, which means they aren't being used anywhere.